src/fluent-plugin-mdsd/lib/fluent/plugin/out_mdsd.rb

```diff
@@ -28,6 +28,8 @@ def initialize()
         config_param :conn_retry_timeout_ms, :integer, :default => 60000
         desc 'the field name for the event emit time stamp'
         config_param :emit_timestamp_name, :string, :default => "FluentdIngestTimestamp"
+        desc 'the maximum record size to emit'
```

Comment:
Let's keep this design for now.

---
In reply to: [125708919](https://github.com/Azure/fluentd-plugin-mdsd/pull/33#discussion_r125708919) [](ancestors = 125708919)